### ISSUE TITLE
What should the final shapes be for calibration?

### ISSUE THREAD

**Issue by komodovaran** [markus93/NN_calibration#6]
It's not entirely clear to me what's going on in the `cal_results` function.

My classifications are of shape (n_samples, n_timesteps, n_classes), which is turned into a 1D array of shape (n_samples, ) by doing argmax on axis -1, which then passes all comparisons between y_true and y_pred.

But the logits are pickled as they are, which leads to a value error in `cal_results`.

I'm not sure I understand correctly. Should they be simply be unravelled, such that

[531, 21, 5, 681, ...] is compared to [1, 0, 0, 1]? Do I need to argmax the logits first?

**Comment by markus93** [comment]
Hey,

sorry for the late response.

I am not sure if it is the issue, but the pickled logits file ((logits_val, y_val), (logits_test, y_test)) should contain the following:
- `logits_val` ( n_val_samples, n_classes) 
- `y_val` (n_val_samples)
- `logits_test` (n_test_samples, n_classes)
- `y_test` (n_test_samples)

Let me know, if You have some more questions.
